resources/assets/js/views/Preview.vue

```diff
@@ -79,6 +79,18 @@
 	<div id="b-overlay" :style="overlayStyles"></div>
 	<resize-shim :onResize="onResize" />
 </div>
+<el-card v-else class="box-card error-card">
+	<div slot="header" class="clearfix">
+		<i class="error-icon el-icon-error is-big"></i>
+		<span class="error-title is-bold">A layout error occured</span>
+	</div>
+	<p>Please report the following errors to your administrator</p>
```

Comment:
Should this message include the current page / site id / path, etc so the user can easily copy and paste for their administrator?

Reply:
good idea